chore: add `.pre-commit-hooks.yaml` (#91)
Enter-tainer/typstyle: README.md

There is an online version of the formatter at <https://enter-tainer.github.io/typstyle/> that you can see how it formats your code.

### Use with [pre-commit](https://github.com/pre-commit/pre-commit)

Add this to your `.pre-commit-config.yaml`:

```yaml
  - repo: https://github.com/Enter-tainer/typstyle

>     rev: ''  # Use the sha / tag you want to point at

Reviewer: ```suggestion
    rev: ''  # The the revision or tag you want to use
```
